grub2/ignition.cfg

```diff
@@ -0,0 +1,15 @@
+# Determine if this is a first boot and set the ${ignition_firstboot} variable
+# which is used in the kernel command line.
+set ignition_firstboot=""
+if [ -f "/ignition.firstboot" ]; then
+    # Default networking parameters to be used with ignition.
```

Comment:
I guess while we're here, might as well `s/ignition/Ignition/` for consistency with the rest of this repo.